Improve the summary of the test report (#241)
karmada-io/website: blog/2022-10-26-test-report/index.md

## Conclusion and Analysis

From the preceding test results, the API Call Latency and Resource Distribution Latency meet the SLIs/SLOs above, and the resources consumed by the system during the whole process are in a controllable range.

> Therefore, Karmada can **stably support 100 large-scale clusters**, manage more than 500,000 nodes and 2 million Pods. In production, Karmada effectively supports the management of hundreds of large-sized clusters.

Reviewer: ```suggestion
As tested, Karmada can **stably support 100 large-scale clusters** running more than 500,000 nodes and 2 million pods, and hundreds of such clusters in production environments.
```

In the process of resource modeling, it will collect node and pod information from all clusters managed by Karmada. This imposes a considerable performance burden in large-scale scenarios.
If you do not use this ability, you can [disable cluster resource modeling](https://karmada.io/docs/next/userguide/scheduling/cluster-resources#disable-cluster-resource-modeling) to further reduce resource consumption.

### Summary and Outlook

> According to the analysis of test results, Karmada can **stably support 100 large-scale clusters**, manage more than 500,000 nodes and 2 million Pods.

Reviewer: ```suggestion
As tested, Karmada can **stably support 100 large-scale clusters** running more than 500,000 nodes and 2 million pods.
```

In terms of use scenarios, Push mode is suitable for managing Kubernetes clusters on public clouds, while Pull mode covers private cloud and edge-related scenarios. In terms of performance and security, the Pull mode outperforms the Push mode.
Each cluster is managed by the karmada-agent component in the cluster and is completely isolated.
However, while the Pull mode improves performance, it also needs to correspondingly improve the performance of karmada-apiserver and karmada-etcd to face challenges in traffic-intensive and high-concurrency scenarios. For specific methods, please refer to the optimization of Kubernetes for large-scale clusters.
In general, users can choose different deployment modes according to usage scenarios to improve the performance of the entire multi-cluster system through parameter tuning and other means.

> Limited by the test environment and test tools, this test has not yet tested the upper limit of Karmada. And the performance test of multi-cluster system is still in the ascendant stage.

Reviewer: ```suggestion
Limited by the environment and tooling, this test does not stress test Karmada. The industry just got started on performance tests of multi-cluster systems.
```

> In the next step, we will continue to optimize the test tools of multi-cluster system and systematically organize the test methods to cover large scales and more scenarios.

Reviewer: ```suggestion
Our next steps will be taken on better tools and methodologies for testing multi-cluster systems in more scenarios.
```
